spec/crypto/asymmetric_spec.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe Asymmetric do
+  let(:keypair) { Asymmetric.generate_new_keypair }
+  let(:private_key) { keypair.to_s }
+  let(:public_key) { keypair.public_key.to_s }
+  let(:team_password) { Symmetric::AES256.random_key }
+
+  it 'should encrypt and decrypt team password' do
+    encrypted_password = Asymmetric.encrypt(team_password, public_key)
```

Comment:
cleartext_data ?

Reply:
now we're abstracting the whole crypto stuff. 
so it doesn't matter what data we're encrypting or decrypting ...
so the whole Crypto Class doesn't have to know about any team passwords